core/src/main/java/org/infinispan/cache/impl/EncoderCache.java

```diff
@@ -471,7 +471,7 @@ public void putAll(Map<? extends K, ? extends V> map, Metadata metadata) {
    @Override
    public CacheSet<CacheEntry<K, V>> cacheEntrySet() {
       EncoderEntryMapper<K, V, CacheEntry<K, V>> cacheEntryMapper = EncoderEntryMapper.newCacheEntryMapper(
-            keyDataConversion, valueDataConversion);
+            keyDataConversion, valueDataConversion, componentRegistry);
```

Comment:
I assumed you were going to pass `entryFactory` here

Reply:
I can do that instead, sure. I saw the other Inject method and forgot those were already handled by EncoderCache.